client/reader/stream/reader-tag-sidebar/index.jsx

```diff
@@ -140,6 +155,25 @@ const ReaderTagSidebar = ( { tag, showFollow } ) => {
 					</Button>
 				</div>
 			) }
+			{ showRecentPrompts && prompts && (
+				<div className="reader-tag-sidebar__recent-prompts">
+					<h2>{ translate( 'Recent Prompts' ) }</h2>
+					{ prompts.map( ( prompt ) => (
+						<div key={ 'prompt-link-' + prompt.id }>
+							<a
+								className="reader-tag-sidebar__recent-prompt-link"
+								href={ '/tag/dailyprompt-' + encodeURIComponent( prompt.id ) }
+								onClick={ () => {
+									handleRecentPromptClick( prompt );
+								} }
+							>
+								{ ' ' }
+								{ prompt.text }{ ' ' }
+							</a>
+						</div>
+					) ) }
+				</div>
+			) }
 		</>
 	);
 };
```

Comment:
```suggestion
};

export default connect( ( state ) => ( {
	primarySiteId: getPrimarySiteId( state ),
} ) )( ReaderTagSidebar );
```

I think you need to get the Primary Site ID for the site ID for blogging prompts - worked for me when I did this at least.

Reply:
ahh ok, thank you!

Reply:
I don't think we need to use `connect` here since we are in a functional component we can just use `getPrimarySiteId` in a `useSelector` above... but either way, I think Eoin is saying we need this primarySiteId to get the prompts, as it looks like the prompts hook depends on that.